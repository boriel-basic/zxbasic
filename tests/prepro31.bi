' Preprocessor bug discovered by LTee

t(x, (y - 1))

