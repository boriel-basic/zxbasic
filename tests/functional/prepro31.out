#line 1 "prepro31.bi"


t(x, (y - 1))

